src/shredder.rs

```diff
@@ -181,6 +182,12 @@ pub trait Shredder {
     /// However, this can be less if the specfic shredder adds some overhead.
     const MAX_DATA_SIZE: usize;
 
+    /// When [`deshred()`] is called, how many data shreds will be produced.
```

Comment:
I mean for `deshred()` this kind of applies as well because of the reconstruction, but that's really secondary
```suggestion
    /// When [`shred()`] is called, how many data shreds will be produced.
```